qcodes/configuration/config.py

```diff
@@ -40,14 +47,14 @@ class Config:
     schema_file_name = "qcodesrc_schema.json"
     """Name of schema file"""
     # get abs path of packge config file
-    default_file_name = str(Path(__file__).parent / config_file_name)
+    default_file_name = str(files("qcodes.configuration") / config_file_name)
```

Comment:
> Note that there is a trade off here as we now have hardcoded the path to config and monitor dist files in the python file rather than the relative path to that file. I think its cleaner to hardcode the full path but I could see the argument the other way around too

would it be possible to use relative imports to address this? e.g. would this work?

```suggestion
    default_file_name = str(files(".") / config_file_name)
```

or perhaps instead of the "dot" there's a builtin variable for "this module where you are now"?

Reply:
It's not documented to work but we give it a try

Reply:
According to stack overflow getting the current module is not something that has a nice api

Reply:
Found out that you can use `__loader__.name` to get the name of the current module. 
This works but is perhaps not too clean. Also ran into https://github.com/python/mypy/issues/4182

Reply:
if it's so not-easy/not-clean to find the path to the current module, then i'm fine with files("qcodes.configuration") for the sake of consistency